python/lsst/qserv/tests/dbLoader.py

```diff
@@ -51,10 +52,19 @@ def __init__(self, config, data_reader, db_name, multi_node, out_dirname):
 
         if self._multi_node:
             self.qAdmin = qservAdmin.QservAdmin('localhost:' + str(self.config['zookeeper']['port']))
-            self.nAdmin = nodeAdmin.NodeAdmin('worker1',
-                                              self.qAdmin,
-                                              wmgrSecretFile=self.config['wmgr']['secret'])
-            self.wmgr = self.nAdmin.wmgrClient()
+            self.nMgmt = nodeMgmt.NodeMgmt(self.qAdmin, wmgrSecretFile=self.config['wmgr']['secret'])
+
+            self.nodeNames = []
+            self.nAdmins = {}
+            self.nWmgrs = {}
+
+            for node in self.nMgmt.selectDict(nodeType='worker', state='ACTIVE'):
+                self.logger.info("Running on node: %s", node)
+                self.nodeNames.append(node)
+                self.nAdmins[node] = nodeAdmin.NodeAdmin(node,
+                                                         self.qAdmin,
+                                                         wmgrSecretFile=self.config['wmgr']['secret'])
+                self.nWmgrs[node] = self.nAdmins[node].wmgrClient()
```

Comment:
I actually forgot to ask you about 'self.nAdmins'. I decided to keep it since I wasn't sure, but once I make the 'wmgr' object from it I no longer need it for anything else correct? If that is the case then we don't have to keep it.